src/main/webapp/resources/css/inwardpayments.css

```
@@ -1,14 +1,14 @@
 @media print{
     .posbill{
-        position: relative!important;  
-        margin-top: -2cm!important;       
+        position: relative!important;
+        margin-top: -2cm!important;
         width: 100%;
         padding: 0px!important;
         margin: 0px!important;
         border: none!important;
         font-family:Arial, Helvetica, sans-serif!important;
         /*        font:*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)